### ISSUE TITLE
Requesting issue closing rights

### ISSUE THREAD

**Issue by Profpatsch** [NixOS/nixpkgs#17379]
I already [did so on the mailing list](http://lists.science.uu.nl/pipermail/nix-dev/2016-July/021177.html), but maybe it’s better to do it here, closer to the point of interest.

Some other people also requested access. @kevincox @vrthra @rasendubi 
@garbas was in favor.

I am not sure who can decide this, so here’s a few cc’s: @edolstra @shlevy  @vcunat @nbp

**Comment by domenkozar** [comment]
Currently you can only request commit access, there's no way to limit to only issues.

**Comment by Profpatsch** [comment]
If that’s so I request commit access (although I won’t use it to push directly).

**Comment by domenkozar** [comment]
cc @rbvermaa

**Comment by vcunat** [comment]
@Profpatsch: I believe there's no reason for you in particular not to push, if you're careful. You've had dozens of PRs accepted over the whole of last year.

**Comment by siddharthist** [comment]
This issue gives me another idea: Maybe we can create a separate issue for writing documentation about under what conditions members of the NixOS community can be granted commit access?

**Comment by vrthra** [comment]
Thanks to @Profpatsch for initiating this request. I too would like to request commit access, and I promise to be very careful.

**Comment by vcunat** [comment]
@siddharthist, there's a related thread: #13602... but the rights themselves aren't that much a problem IMHO.

**Comment by rasendubi** [comment]
Thanks, @Profpatsch. I would also like to have write access. I promise I won't use it for anything except issue management.

**Comment by rbvermaa** [comment]
@grahamc is busy with open sourcing a tool that will allow some issue management without needing push access (via the comments). Once that is done, we'll enable that. Please be patient for another week or so.

**Comment by matthewbauer** [comment]
Another related issue is labels, you need commit access to be able to add labels to an issue/pr.

**Comment by 7c6f434c** [comment]
@rbvermaa A month has passed. I think @grahamc is busy with other things…

It also looks that in general @Profpatsch and @rasendubi contribute code regularly anyway.

I vote for giving them  write access now.

**Comment by vrthra** [comment]
@7c6f434c, I hope I am not pushing my case too much, but I do have a reasonable number of accepted PRs if the amount of contribution is a criterion. If my candidacy does not make the cut, could I also have some hint of what factors goes into the decision so that I can work to make it happen?

```
$ git shortlog -s | sort -rn | nl | grep '\(Profpatsch\|Shmalko\|Rahul\)'
    82     124  Rahul Gopinath
   117      70  Profpatsch
   152      44  Alexey Shmalko
```

**Comment by 7c6f434c** [comment]
@vrthra Oops, too much thread derailing to skim it correctly, sorry. @rbvermaa yes, I also support giving commit rights to @vrthra .

**Comment by 7c6f434c** [comment]
@vrthra Oops, too much thread derailing to skim it correctly, sorry. @rbvermaa updated the recommendation list

**Comment by grahamc** [comment]
Hey everyone, I just wanted to post an update. I'm sorry a month has passed since.

 - I've had a hard time open sourcing the other tool, I thought it would be easier
 - I started writing it again from scratch, but the next two confounded that a bit 
 - I started a new job
 - I moved across the country

so in a lot of ways I have been busy with other things. I'll post another update shortly.

**Comment by 7c6f434c** [comment]
>Hey everyone, I just wanted to post an update. I'm sorry a month has passed since.

Well, it's fine, life happens. I am sorry I have failed to keep the
balance between not putting pressure on you and giving arguments for the
fallback solution to be applied right now.

> - I started a new job
> - I moved across the country

I do not know the circumstances so I cannot decide whether the former or
the latter merits congratulations. Congratulations with whichever of 
these two things was the driving reason, though.

**Comment by grahamc** [comment]
>  I am sorry I have failed to keep the
> balance between not putting pressure on you and giving arguments for the
> fallback solution to be applied right now.

No worries, these are all just excuses anyway.

> I do not know the circumstances so I cannot decide whether the former or
> the latter merits congratulations. Congratulations with whichever of
> these two things was the driving reason, though.

I'll take both. Thank you!

**Comment by shlevy** [comment]
@edolstra @rbvermaa Can I have commit-right-granting rights?

**Comment by domenkozar** [comment]
I've given commit access to @Profpatsch, @rasendubi @kevincox and @vrthra.

So far the policy has been to give access to people that have already contributed and asked for it.

Please be careful when you push changes to master as it effects a lot of people. Thank you for being part of the community!

**Comment by domenkozar** [comment]
Oh and sorry for the slow process. We'll have an official way to request commit access one day and I'm sure that will make it easier.

**Comment by shlevy** [comment]
@domenkozar In the mean time can we document who to ping in README + manual?

**Comment by domenkozar** [comment]
Sure, let's document it. README is not the right place, not sure if manual also.

Maybe CONTRIBUTING.md?
